go/obscuronode/enclave/sql/edbattestation.go

```diff
@@ -0,0 +1,204 @@
+package sql
+
+// todo: was hoping to use the edgeless tool ERA as a dependency for verifying the edgeless db attestations but I'm hitting
+//   dependency issues from `eclient` so progressing by duplicating relevant part of ERA using their `enclave` client
+//   library instead of `eclient`
+
+// ERA (Edgeless remote attestation) is a simple protocol for verifying edgeless tools. It's basically just a small json schema
+// that they use as a standard data blob to encrypt into their attestation reports, includes signerID, security version etc.
+
+// The only changes from https://github.com/edgelesssys/era/blob/master/era/era.go is the use of enclave.VerifyRemoteReport
+// in place of eclient.VerifyRemoteReport and a few tweaks to get our lint to pass.
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"crypto/tls"
+	"encoding/binary"
+	"encoding/hex"
+	"encoding/json"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/edgelesssys/ego/attestation"
+	"github.com/edgelesssys/ego/attestation/tcbstatus"
+	"github.com/edgelesssys/ego/enclave"
+	"github.com/tidwall/gjson"
+)
+
+type certQuoteResp struct {
+	Cert  string
+	Quote []byte
+}
+
+// ErrEmptyQuote defines an error type when no quote was received. This likely occurs when the host is running in OE Simulation mode.
+var ErrEmptyQuote = errors.New("no quote received")
+
+// GetCertificate gets the TLS certificate from the server in PEM format. It performs remote attestation
```

Comment:
@BedrockSquirrel Can you clarify in the comment what server you're referring to here?